rest/route/select.go

```diff
@@ -0,0 +1,72 @@
+package route
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+
+	"github.com/evergreen-ci/gimlet"
+	"github.com/evergreen-ci/utility"
+	"github.com/mongodb/grip"
+	"github.com/pkg/errors"
+)
+
+type selectTestsHandler struct {
+	selectTests SelectTestsRequest
+}
+
+func makeSelectTestsHandler() gimlet.RouteHandler {
+	return &selectTestsHandler{}
+}
+
+// Factory creates an instance of the handler.
+//
+//	@Summary		Select tests
+//	@Description	Return a subset of tests to run for a given task. This endpoint is not yet ready. Please do not use it.
```

Comment:
Should we link a ticket here? I only fear that it will have this description forever

Just as a heads-up, I am a secondary review ATM (and I am new to it) which means another person on evg-app would need to be a primary reviewer and if you have any pointers, suggestions, concerns, etc, feel free to reply back or dm please!

Reply:
Unfortunately there isn't an existing ticket for making this work. In general it will be DevProd who will be implementing against this interface, so I think it's okay if this is here indefinitely.